pkg/sidecar-controller/framework_test.go

```diff
@@ -522,6 +523,7 @@ func newTestController(kubeClient kubernetes.Interface, clientset clientset.Inte
 		"snapshot",
 		-1,
 		true,
+		workqueue.NewItemExponentialFailureRateLimiter(1*time.Second, 5*time.Minute),
```

Comment:
This is a test so it's better to set the time shorter.  Suggest changing to the following:
1*time.Second -> 1*time.Millisecond
5*time.Minute -> 1*time.Minute

Reply:
@xing-yang done. thanks.. ptal.